pubspec.yaml

```
@@ -7,13 +7,11 @@ environment:
   sdk: ^3.8.0
 
 dependencies:
-  kiss_repository:
-    path: ../kiss_repository
+  kiss_repository: ^0.11.0 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)